packages/volto/src/components/theme/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -51,6 +51,10 @@ const BreadcrumbsComponent = ({ pathname }) => {
     }
   }, [dispatch, pathname]);
 
+  if (pathname.endsWith('/uploads')) {
```

Comment:
there has to be a better way to avoid rendering certain components based on some route.
Try https://training.plone.org/effective-volto/addons/views.html#router-views
to add uploads to nonContentRoutes and see if this doesn't solve the content rendering

Reply:
nope this didn't work. all the contents are rendered under app.jsx where the header, breadcrumbs and footer components are static and the content-area is dynamically rendered based on the route. 
https://github.com/plone/volto/blob/main/packages/volto/src/components/theme/App/App.jsx, maybe we could move the logic the app.jsx file instead of having it seperately in header, footer and breadcrumbs file.